usecases/modulecomponents/settings/base_class_settings.go

```diff
@@ -27,6 +28,14 @@ const (
 	DefaultVectorizePropertyName = false
 )
 
+var errInvalidProperties = fmt.Errorf("invalid properties: didn't find a single property which is " +
+	"of type string or text and is not excluded from indexing. In addition the " +
+	"class name is excluded from vectorization as well, meaning that it cannot be " +
+	"used to determine the vector position. To fix this, set 'vectorizeClassName' " +
+	"to true if the class name is contextionary-valid. Alternatively add at least " +
+	"contextionary-valid text/string property which is not excluded from " +
+	"indexing")
```

Comment:
should we change the text a bit? Very much contextionary focused

Reply:
updated message